cmd/influx/backup.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/bolt"
+	"github.com/influxdata/influxdb/http"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/multierr"
+)
+
+var backupCmd = &cobra.Command{
+	Use:   "backup",
+	Short: "Backup the data in InfluxDB",
+	Long: fmt.Sprintf(
+		`Backs up data and meta data for the running InfluxDB instance.
+Downloaded files are written to the directory indicated by --path.
+The target directory, and any parent directories, are created automatically.
+Data file have extension .tsm; meta data is written to %s in the same directory.`,
+		bolt.DefaultFilename),
+	RunE: backupF,
+}
+
+var backupFlags struct {
+	Path string
+}
+
+func init() {
```

Comment:
can you add some tests here and remove the globals in line with hte direction were heading in the `influx` pkg.  See here for an example of testing in this pkg:
https://github.com/influxdata/influxdb/pull/16491/commits/ee3198fb07ebcd01500c52c19f57b3af257493e0

Reply:
Thanks for bringing this up. Looks like you merged just today, so when I rebased for another reason, this fix became necessary. Please take another look.